docsrc/conf.py

```diff
@@ -1183,14 +1183,14 @@
 
     #You are looking at documentation that is maintained by interval.
 
-    #Please see https://docs.cyrus.foundation/ for better maintained
+    #Please see https://www.cyrusimap.org/ for better maintained
     #documentation.
 #"""
 
 # Use this as :task:`18`
 extlinks = {
         'rfc':('http://tools.ietf.org/html/rfc%s', 'RFC '),
-        'task':('https://git.cyrus.foundation/T%s', 'Task #'),
+        'task':('https://github.com/cyrusimap/cyrus-imapd/issues/%s', 'Task #'),
```

Comment:
Ah, true. I was hoping that Brong's magic autoimporter would let us identify phab/bugzilla issues in github, but perhaps that was somewhat... wishful.

I'll leave it pointing to Phab for now and make a new :issue: extlink. When everything gets merged, we can revisit this.